chef_master/source/aws_marketplace.rst

```diff
@@ -149,9 +149,9 @@ Launch the BYOL AMI
 #. Associate the IAM role for backup access.
 #. Run the CloudFormation template to create the Chef Automate instance.
 
-Install the Chef DK
+Install the ChefDK
```

Comment:
Should this be workstation?

Reply:
No, because it references Automate (see line 141). You said to stay away from Automate docs, right?

Reply:
@jonsmorrow The better question, I think, is does this information still apply in A2 with Workstation also? Or just A1?

Reply:
fixed! It's still used.